01-path-basics/102-your-first-cluster/readme.adoc

```diff
@@ -1,127 +1,96 @@
+This will create a configuration file at `$HOME/.kube/config-k8s-workshop` and update the necessary environment variable for default access.
 
-Your cluster example.cluster.k8s.local is ready
-```
+=== Create the worker nodes
 
-anchor:multi-master[]
+Now that your EKS master nodes are created, you can launch and configure your worker nodes.
 
-=== Multi-Master Cluster
+To launch your worker nodes, run the following CloudFormation CLI command:
 
-The command below creates a cluster in a multi-master, multi-node, and multi-az configuration.
-Run it in the "bash" terminal tab at the bottom of the Cloud9 IDE.
-We can create and build the cluster in one step by passing the `--yes` flag.
+    $ aws cloudformation create-stack \
+      --stack-name k8s-workshop-worker-nodes \
+      --template-url https://amazon-eks.s3-us-west-2.amazonaws.com/1.10.3/2018-06-05/amazon-eks-nodegroup.yaml \
+      --capabilities "CAPABILITY_IAM" \
+      --parameters "[{\"ParameterKey\": \"KeyName\", \"ParameterValue\": \"${AWS_STACK_NAME}\"},
+                     {\"ParameterKey\": \"NodeImageId\", \"ParameterValue\": \"${EKS_WORKER_AMI}\"},
+                     {\"ParameterKey\": \"ClusterName\", \"ParameterValue\": \"k8s-workshop\"},
+                     {\"ParameterKey\": \"NodeGroupName\", \"ParameterValue\": \"k8s-workshop-nodegroup\"},
+                     {\"ParameterKey\": \"ClusterControlPlaneSecurityGroup\", \"ParameterValue\": \"${EKS_SECURITY_GROUPS}\"},
+                     {\"ParameterKey\": \"VpcId\", \"ParameterValue\": \"${EKS_VPC_ID}\"},
+                     {\"ParameterKey\": \"Subnets\", \"ParameterValue\": \"${EKS_SUBNET_IDS}\"}]"
 
-    $ kops create cluster \
-      --name example.cluster.k8s.local \
-      --master-count 3 \
-      --node-count 5 \
-      --zones $AWS_AVAILABILITY_ZONES \
-      --yes
+The `AWS_STACK_NAME`, `EKS_WORKER_AMI`, `EKS_VPC_ID`, `EKS_SUBNET_IDS`, and `EKS_SECURITY_GROUPS` environment variables should have been set during the link:../101-start-here[Cloud9 Environment Setup].
 
-A multi-master cluster can be created by using the `--master-count` option and specifying the number of master nodes. An odd value is recommended. By default, the master nodes are spread across the AZs specified using the `--zones` option. Alternatively, you can use the `--master-zones` option to explicitly specify the zones for the master nodes.
+Node provisioning usually takes less than 5 minutes. You can query the status of your cluster with the following command. When your cluster status is `CREATE_COMPLETE`, you can proceed.
 
-The `--zones` option is also used to distribute the worker nodes. The number of workers is specified using the `--node-count` option.
+    $ aws cloudformation describe-stacks --stack-name k8s-workshop-worker-nodes --query 'Stacks[0].StackStatus' --output text
 
-It will take 5-8 minutes for the cluster to be created. Validate the cluster:
+=== Enable worker nodes to join cluster
 
-```
-$ kops validate cluster
```

Comment:
Can we add a `aws eks describe-cluster` as a validation step?

Reply:
It's already in there on line 33.  We use it to monitor the cluster creation.

    $ aws eks describe-cluster --name k8s-workshop --query cluster.status --output text